### PR TITLE
Add brotli/1.0.7 recipe

### PR DESCRIPTION
**Pull request by madebr**
Specify library name and version:  **brotli/1.0.7**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by uilianries** [comment]
@madebr please take a look: https://github.com/madebr/conan-center-index/pull/3

**Comment by madebr** [comment]
> @madebr please take a look: [madebr#3](https://github.com/madebr/conan-center-index/pull/3)

Thanks! I have added it.

File: recipes/brotli/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+    def _get_libraries(self, shared):
+        libs = ["brotlienc", "brotlidec", "brotlicommon"]
+        if not shared:
+            libs = ["{}-static".format(l) for l in libs]
+        return libs
+
+    def package_info(self):
+        self.cpp_info.name = "Brotli"
```
Comment (danimtb) [inline recipes/brotli/all/conanfile.py]: Use specific generators
```suggestion
        self.cpp_info.names["cmake_find_package"] = "Brotli"
        self.cpp_info.names["cmake_find_package_multi"] = "Brotli"
```

File: recipes/brotli/all/test_package/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
```
Comment (danimtb) [inline recipes/brotli/all/test_package/conanfile.py]: Can we use here just the cmake_find_package generator? I would like to have some test packages only relying on that one for testing purposes. Thanks!

**Pull request merged**